metafix/src/test/resources/org/metafacture/metafix/integration/record/fromJson/toJson/set_arrayNestedSimple/test.fix

```diff
@@ -0,0 +1,2 @@
+set_array("object.test[]")
+set_array("object_2.test_2[]", "dog", "cat", "bird")
```

Comment:
Should probably be two tests.

Reply:
I have split the `set_array` tests. Found an error and added some more variations of the tests.